scripts/functional-tests.sh

```diff
@@ -118,8 +121,8 @@ cleanup() {
     local original_model_filename="${original_model_name}.gguf"
     local temporary_model_filename='foo.gguf'
     sed -i.bak "s/baz/${original_model_name}/g" "${ILAB_CONFIG_FILE}"
-    mv "${ILAB_DATA_DIR}/models/${temporary_model_filename}" "${ILAB_DATA_DIR}/models/${original_model_filename}" || true
```

Comment:
(No action required) nit: You could put the full path (`${ILAB_DATA_DIR}/models/${temporary_model_filename}`) in a variable, then you wouldn't need to construct it twice here.

Reply:
Done!